modules/apps/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/display/UADHierarchyDeclaration.java

```diff
@@ -40,6 +40,12 @@
 	 */
 	public UADDisplay<?>[] getContainerUADDisplays();
 
+	/**
+	 * Returns the label that is displayed in the Entities Type Panel.
+	 *
+	 * @param  locale the label's retrieved locale
+	 * @return the label of the Entities Type Panel entry
+	 */
```

Comment:
- [ ] remove reference to "Entities Type Panel"
- [ ] describe what the label should be describing